oauthenticator/tests/test_globus.py

```diff
@@ -0,0 +1,35 @@
+from pytest import fixture, mark
+
+from ..globus import GlobusOAuthenticator
+
+from .mocks import setup_oauth_mock, no_code_test
+
+
+def user_model(username):
+    """Return a user model"""
+    return {
+        'login': username,
+    }
+
+@fixture
+def globus_client(client):
+    setup_oauth_mock(client,
+        host=['github.com', 'api.github.com'],
+        access_token_path='/login/oauth/access_token',
+        user_path='/user',
```

Comment:
I'm more in favor of using the SDK lib than not using it ( :open_mouth: ), as a general rule, but it's not obvious that it will actually make your life easier here.

You can use the lib as a reference and go straight to the [API docs](https://docs.globus.org/api/auth/reference/) -- as far as Globus is concerned, integrating directly against this API is totally fine.

We only built a lib because we consider OAuth2 to be unnecessary knowledge for most people developing against our APIs. Clearly, building this integration requires decent familiarity with OAuth2, so it's not obvious that doing things directly is in any way inferior.